clang/test/SemaSYCL/sycl-pfwg-invalid-code.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang_cc1 -fsycl-is-device %s -verify
+
+// Tests that an error diagnostic is issued without a crash (due to a
```

Comment:
I'm not sure the change is functional. Also, it doesn't really add a diagnostic, so I'm not sure the comment is quite true. I would prefer something like "Tests that the compiler doesn't crash if definition of `kernel_parallel_for_work_group` is not valid".

Reply:
Thanks!